diffyscan/diffyscan.py

```diff
@@ -106,6 +106,18 @@ def run_source_diff(
     recursive_parsing=False,
     prettify=False,
 ):
+    source_files = (
+        contract_code["solcInput"].items()
+        if "sources" not in contract_code["solcInput"]
+        else contract_code["solcInput"]["sources"].items()
+    )
+
+    if is_monolithic_contract(source_files):
```

Comment:
I'd suggest to shape the func checking if the contract machine-friendly to verify, see: https://docs.soliditylang.org/en/latest/using-the-compiler.html#compiler-input-and-output-json-description 

```suggestion
    if not is_standard_json_contract(source_files):
```